packages/dds/tree/src/test/sequence-change-family/testGeneratedChanges.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { strict as assert } from "assert";
+import * as fs from "fs";
+import { toDelta, Transposed as T } from "../../changeset";
+import { sequenceChangeRebaser } from "../../feature-libraries";
+import { verifyChangeRebaser } from "../../rebase";
+import { FieldKey } from "../../tree";
+import { brand } from "../../util";
+import { generateFuzzyCombinedChange } from "../rebase/fuzz";
+import { generateRandomChange, generateRandomUpPath } from "./randomSequenceGenerator";
+
+const fooKey = brand<FieldKey>("foo");
+const keySet = new Set([fooKey]);
+
+const pathGen = (seed: number) => generateRandomUpPath(keySet, seed, 3, 5);
+describe("testing the SequenceChangeRebaser", () => {
```

Comment:
```suggestion
describe("SequenceChangeRebaser - Fuzz", () => {
```
The "testing the" part is implicit, so it is generally omitted.
I think adding the "Fuzz" part is helpful since there are other tests for it (in `sequenceChangeRebaser.spec.ts`).
On that note, I think it would be good to rename the file to `sequenceChangeRebaser.fuzz.spec.ts`